### PR TITLE
189 update ug register account

### LINKED ISSUE #168: Update UG - Account Recovery

### LINKED ISSUE #169: Update register account for account recovery
Missing fields such as secretQn and answer

### LINKED ISSUE #189: Update UG - Register Account

### PR DESCRIPTION
**Pull request by jianyangg**
Close #189.
Close #168.
Close #169.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -95,17 +100,42 @@ _Details coming soon..._
...
 
-### Change Password `[Coming Soon in v1.3]`
+### Account Recovery
 
-_Details coming soon..._
+**Format:** `recover account [--answer ANSWER --password NEW_PASSWORD --confirmPass CONFIRM_PASSWORD]`
 
-### Update details `[Coming Soon in v1.3]`
+**Example 1:** `recover account`
+**Example:** `recover account [--answer Koko --password newPassword123 --confirmPass newPassword123]`
```
Comment (zhonghan721) [inline docs/UserGuide.md]: Would it be `Example 2` and on a new line? Also, would it be better not to show `[` and `]` as it is an example.

**Review by zhonghan721 — COMMENTED** [review]
Some general feedback.

File: docs/UserGuide.md (hunk 1 above)
Comment (jianyangg) [reply docs/UserGuide.md]: Good spot, thanks!

**Review by zhonghan721 — APPROVED** [review]
LGTM

**Pull request merged**